static/src/javascripts/bootstraps/enhanced/notifications.js

```diff
@@ -43,37 +48,65 @@ define([
             return modules.getReg().then(function (reg) {return reg.pushManager.getSubscription();});
         },
 
-        configureSubscribeTemplate: function () {
-            var subscribed = modules.checkSubscriptions(),
-                hasNoSubscriptions = modules.subscriptionsEmpty(),
+        init: function() {modules.configureSubscribeButton();
+            if(!modules.hasSubscribed() && !modules.hasDismissedExplainer()) {
+                modules.showExplainer();
+            }
+        },
+
+        configureSubscribeButton: function () {
+            var $follow = bonzo(qwery('.js-live-notifications')[0]),
+                subscribed = modules.checkSubscriptions(),
                 handler = subscribed ? modules.unSubscribeHandler : modules.subscribeHandler,
-                src = template(subscribeTemplate, {
-                    className: hasNoSubscriptions ? '' : 'notifications-subscribe-link--has-subscriptions',
-                    text: subscribed ? 'Unfollow' : 'Follow story',
-                    imgMobile: svgs('notificationsExplainerMobile', ['mobile-only', 'notification-explainer']),
-                    imgDesktop: svgs('notificationsExplainerDesktop', ['hide-on-mobile', 'notification-explainer'])
+                src = template(followLink, {
+                    subscribed: subscribed,
+                    icon: svgs(subscribed ? 'notificationsOff' : 'notificationsOn')
                 });
 
+            $follow.html(src);
+            bean.one($follow[0], 'click', '.js-notifications__button', handler );
+
+        },
+
+        showExplainer: function() {
+            var src = template(explainer,{
+                closeIcon : svgs('closeCentralIcon'),
+                imgMobile: svgs('notificationsExplainerMobile', ['mobile-only', 'live-notifications-explainer-svg']),
+                imgDesktop: svgs('notificationsExplainerDesktop', ['hide-on-mobile', 'live-notifications-explainer-svg'])
+            });
+
             fastdom.write(function () {
-                $('.js-notifications').prepend(src);
-                bean.one(document.body, 'click', '.js-notifications-subscribe-link', handler);
+                var notifications = $('.js-live-notifications');
+                notifications.append(src);
+                bean.one(notifications[0], 'click', '.js-live-notifications__item__close', function() {
```

Comment:
You probably don't need two levels of nesting in the BEM classname here? Will `.js-live-notifications__item__close` suffice?